public/ConvertFrom-FunctionFile.ps1

```diff
@@ -0,0 +1,52 @@
+﻿function ConvertFrom-FunctionFile {
+    <#
+        .Synopsis
+            Exports all functions from a PS*1 file and stores them in separate function files
+        
+        .Description
+            Using AST, extracts all function definitions from specified files, and create
+            a file per function within the module directory provided. This does not currently
+            rewrite the specified file to import the module, nor remove the existing functions.
```

Comment:
Nor does it create a `prefix.ps1` or `postfix.ps1` file with all the non-function code, nor does it even warn you that such code exists and is being ignored ...

Reply:
I've rewritten this to be a lot less specific about the things it doesn't do. ;) We can assume that we're only extracting functions, currently.

"nor does it cure world hunger"